### DISCUSSION TITLE
I am waiting for the student pack in vain

### DISCUSSION THREAD

**Discussion by TranToan097** [community/community#140093] [GitHub Education]
### Hi everyone,

It's been 3 days now and I still haven't received it :(
![Screenshot_2024-09-29-16-03-30-192_com android chrome](https://github.com/user-attachments/assets/5bc8775d-6a95-4ae6-b5c6-8cae6f5f8d14)

**Comment by davevad93** [comment]
Hi @TranToan097 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @TranToan097 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
